core/src/main/res/values/strings.xml

```diff
@@ -842,4 +838,11 @@
     <string name="on_demand_config_setting_changed">Setting updated successfully.</string>
     <string name="on_demand_config_stream_text">Looks like you stream a lot. Do you want episode lists to show stream buttons?</string>
     <string name="on_demand_config_download_text">Looks like you download a lot. Do you want episode lists to show download buttons?</string>
+    <string name="preference_synchronization_summary_unchoosen">You can choose from multiple providers to synchronize your subscriptions and episode play state with</string>
+    <string name="preference_synchronization_summary_nextcloud">Gpoddersync app which needs to be installed on a nextcloud server. Both of which are independent of the AntennaPod project</string>
```

Comment:
How about a small wording change to make it more consistent with the gpodder description?

```suggestion
    <string name="preference_synchronization_summary_nextcloud">Gpoddersync is an open-source Nextcloud app that you can easily install on your own server. The app is independent of the AntennaPod project.</string>
```

Reply:
I don't feel any of them shortly explains what this is. It is just too many components interacting 😄 
Let's take yours then we have at least one who is satisfied with the description